editors/sc-ide/SuperColliderIDE.desktop

```diff
@@ -1,9 +1,11 @@
 [Desktop Entry]
 Name=SuperCollider IDE
 GenericName=SuperCollider IDE
+Comment=Audio synthesis and algorithmic musical creation
```

Comment:
maybe better: "algorithmic music"

Reply:
it was "musical composition" before, but I though that was too composition oriented. Simply "music" is better (it's too long as it is). What about "algorithmic sound"?

Reply:
"Editor for the SuperCollider sound synthesis language"?

Reply:
Duh... of course!

Reply:
should it be audio or sound synthesis? also, is SC a just language or is it an environment or a framework? I'm never sure how to describe the project as a whole.

Reply:
audio/sound are the same to me.

i tend to succinctly describe SC as "a sound synthesis platform." you can do "editor for the SuperCollider sound synthesis platform" if you want, but i chose the wording because the editor is specifically for editing sclang.

Reply:
ok - going with "IDE for the SuperCollider audio synthesis language". I think we say "audio synthesis" more than "sound synthesis".